src/components/trade/PriceSuggestions/PriceSuggestions.stories.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from 'react'
+import { useForm, FormProvider } from 'react-hook-form'
+
+// also exported from '@storybook/react' if you can deal with breaking changes in 6.1
+import { Meta, Story } from '@storybook/react/types-6-0'
+import { PriceSuggestions, Props } from './PriceSuggestions'
+import { Frame } from 'components/common/Frame'
+import BigNumber from 'bignumber.js'
+
+export default {
+  title: 'Trade/PriceSuggestions',
+  components: PriceSuggestions,
```

Comment:
Book confused
There is `component` and there is `subcomponents`, no `components`

Reply:
I'll address with the other ones